template/nav.html

```diff
@@ -45,27 +45,15 @@
 
             <!-- Search, Navigation and Repo links -->
             <ul class="nav navbar-nav navbar-right">
-                {% if include_next_prev %}
```

Comment:
Instead of deleting this block, can we replace the check by `{% if config.include_next_prev %}` (and keep `include_next_prev: false` in `mkdocs.yml`)?

Reply:
👍